modules/persistenceService/src/main/resources/db/migration/V3__ADD_SUBS_AT_FIELD.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE subscriptions
+ADD COLUMN subscribed_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP;
```

Comment:
```suggestion
ADD COLUMN subscribed_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP;
```
Or work with `Option` if you want to make the column nullable.